paddle/fluid/framework/ir/mkldnn/cpu_quantize_pass_tester.cc

```diff
@@ -417,36 +417,88 @@ void TestImmutableOpBetweenNonQuantizedOp(const std::string tested_op) {
            SCALE * S8_MAX);
 }
 
+// a->Dropout1->b
+// b->TestedOp1(not quantized)->c
```

Comment:
Yes, because in `cpu_quantize_placement_pass` we set mkldnn_data_type to int8 to all supported operators, then in `cpu_quantize_pass` we decide if it will be quantized or not. Maybe I should change this comment to `(will be quantized)` and `(won't be quantized)`

Reply:
Sounds good. Or should/shouldn't be quantized sounds also good.